.sqlfluff

```diff
@@ -33,13 +25,12 @@ exclude_rules = aliasing.column,
     references.from,
     references.keywords,
     references.qualification,
-    references.quoting,
     structure.column_order,
     structure.distinct,
+    # While explicit, this provides clarity.
     structure.else_null,
     structure.join_condition_order,
-    structure.simple_case,
-    structure.subquery,
+    # False positive -- see https://github.com/sqlfluff/sqlfluff/pull/7095
     structure.unused_cte,
     structure.unused_join,
```

Comment:
unused_join has 1 very strange false positive as well, it's joining a select result with an update query to update all matching.